catalog/core/catalog-core-standardframework/src/main/java/ddf/catalog/impl/operations/QueryOperations.java

```diff
@@ -438,9 +446,12 @@ private QueryResponse processPostQueryPlugins(QueryResponse queryResponse)
 
   private QueryResponse processPostQueryAccessPlugins(QueryResponse queryResponse)
       throws FederationException {
+    String traceId = ThreadContextProperties.getTraceId();
     for (AccessPlugin plugin : frameworkProperties.getAccessPlugins()) {
       try {
+        LOGGER.trace("Before AccessPlugin (#2): trace-id {}, plugin {}", traceId, plugin);
```

Comment:
✏️ Perhaps it would be a little clearer in the logs if we use "PostQuery" and "PreQuery" instead of "#2" and "#1", respectively.  Otherwise, you would have to look at the code to know what 1 and 2 mean.  This would be true for all of the following log statements as well.

Reply:
done